Microsoft.Alm.Authentication/Src/Network.cs

```diff
@@ -552,6 +552,7 @@ private HttpClient GetHttpClient(TargetUri targetUri, HttpMessageHandler handler
                             switch (token.Type)
                             {
                                 case TokenType.AzureAccess:
+                                case TokenType.BitbucketAccess:
```

Comment:
Make sure the BitbucketAccess creates a Bearer token authentication header

Reply:
:+1: